lib/rubocop/cop/betterment/boolean_query_parameter.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Betterment
+      class BooleanQueryParameter < Base
+        MSG = <<~MSG
+          A boolean fetched from query params will never be false when
+          explicitly specified on the request. Please use
+          ActiveModel::Type::Boolean.new.cast to cast the value to a boolean.
+        MSG
+
+        # @!method fetch_boolean?(node)
+        def_node_matcher :fetch_boolean?, <<-PATTERN
+          (send
+            (send ...) :fetch
+            (sym ...)
+            (boolean))
+        PATTERN
+
+        # @!method boolean_cast?(node)
+        def_node_search :boolean_cast?, <<-PATTERN
+          (send
+            (send
+              (const
+                (const
+                  (const nil? :ActiveModel) :Type) :Boolean) :new) :cast
+            ...)
+        PATTERN
+
+        # @!method action_controller?(node)
+        def_node_search :action_controller?, <<~PATTERN
+          {
+            (const {nil? cbase} :ApplicationController)
+            (const (const {nil? cbase} :ActionController) :Base)
+          }
+        PATTERN
+
+        def initialize(config = nil, options = nil)
+          super(config, options)
```

Comment:
__suggestion:__ This could be `super`, and it'll default to this.

Reply:
I think I can get rid of this whole method. Tests pass without it, just an artifact of copying the skeleton from another cop. Done